images/scripts/rhel.setup

```diff
@@ -168,6 +165,9 @@ virt-install \
 
 if [ "${IMAGE#rhel*}" != "$IMAGE" ]; then
     TEST_PACKAGES="${TEST_PACKAGES} subscription-manager-cockpit insights-client"
+    if [ "$IMAGE" != "rhel-8-3" ]; then
+        TEST_PACKAGES="${TEST_PACKAGES} python3-tracer kpatch kpatch-dnf"
```

Comment:
kpatch sounds like it may indeed be RHEL specific, as that's quite a value-added service. tracer OTOH sounds like it should flow into centos at some point. I normally prefer these quirks the other way round, ie. `if image == rhel-8-3` then `drop these packages from TEST_PACKAGES. But we can switch this around the next time for sure.

Reply:
right, I even think that tracer reached centos in https://github.com/cockpit-project/bots/pull/1861
Let me turn it around then.